packages/govuk-frontend/src/govuk/components/file-upload/file-upload.yaml

```diff
@@ -10,7 +10,9 @@ params:
   - name: value
     type: string
     required: false
-    description: Deprecated. Optional initial value of the input.
```

Comment:
Potentially we could JUST have one field, something like: `deprecated: '5.7.1'`?

Reply:
Just trying to think of a case where something isn't specifically deprecated in a version. Struggling though!

Reply:
Could it accept either a string or a boolean, and alter the output depending on whether a string is present or not?

Reply:
The only thing about that is that we don't seem to define a macro option schema, and this would actually only affect output on the DS website, so not very clear here what the field accepts.

I guess the answer is: Well then, define a macro option schema, doofus.